civictechprojects/urls.py

```diff
@@ -77,7 +77,7 @@
     re_path(r'^api/groups', views.group_search),
     re_path(r'^api/tags/groups', views.group_tags_counts),
     re_path(r'^api/tags', views.tags),
-    re_path(r'^admin/', admin.site.urls),
+    re_path(r'^secretadmin/', admin.site.urls),
```

Comment:
If we're going to do this, the path should be configurable in a non-public place, such as an environment variable.  As it is, any attackers can simply look at the source code and see exactly where they need to go.

Reply:
good point. Will work on this.